### ISSUE TITLE
Error while signin

### ISSUE THREAD

**Issue by mash420** [rongardF/tvdatafeed#26]
Yesterday, I was able to use my credentials successfully. I received emails from tradingview saying new login with Python etc...
Today I am getting the error while sign in notification when requesting data. 
How do I fix this?

**Comment by JustKovalchuk** [comment]
Same issue. @mash420 have you fixed it? 
I suppose that it has some delay for logging in. Because it worked during few minutes another day. So if you log in a lot of times you will be banned

**Comment by mash420** [comment]
Nope, I imagine tradingview blocked it. It worked the first day, then
stopped working altogether since then.

**Comment by hellmut92** [comment]
As I can see, when I download data with credentials, Tradingview authorize 3 to 4 data request (each time sending a "new login" email) and then block any request for 24 / 36 hours.

These connections can be seen in the "Settings/privacy and security" section of web version of tradingview as an unknown device with an unknown OS with location and IP address and under the name "python Request 2.31"

I guess that to make Tradingview happy with the connections would be to accept cookies like a web browser would do...

**Comment by mash420** [comment]
any fix?

**Comment by hellmut92** [comment]
No, I gave up and subscribed to FMP.

**Comment by jamezun** [comment]
I am having this error as well:
error while signin
you are using nologin method, data you access may be limited

I am a premium user, I expect I could have access more data but was limited

**Comment by hellmut92** [comment]
> I am having this error as well: error while signin you are using nologin method, data you access may be limited
> 
> I am a premium user, I expect I could have access more data but was limited

I'm not a premium user, but i got the quotes back. Maybe there is an issue with you login/passwrd.

Anyway the problem is when you use a login, tradingview see it as a security breach and is going to stop to respond after 4/5 attempt.

**Comment by jamezun** [comment]
> > I am having this error as well: error while signin you are using nologin method, data you access may be limited
> > I am a premium user, I expect I could have access more data but was limited
> 
> I'm not a premium user, but i got the quotes back. Maybe there is an issue with you login/passwrd.
> 
> Anyway the problem is when you use a login, tradingview see it as a security breach and is going to stop to respond after 4/5 attempt.

yes, somehow the error was gone now, thanks!

**Comment by andreasklinar** [comment]
> > > I am having this error as well: error while signin you are using nologin method, data you access may be limited
> > > I am a premium user, I expect I could have access more data but was limited
> > 
> > 
> > I'm not a premium user, but i got the quotes back. Maybe there is an issue with you login/passwrd.
> > Anyway the problem is when you use a login, tradingview see it as a security breach and is going to stop to respond after 4/5 attempt.
> 
> yes, somehow the error was gone now, thanks!

Did you do anything to resolve it or did it just resolve itself after a few days?

**Comment by gopal86** [comment]
@jamezun @andreasklinar were you able to resolve the error? As I am still getting the same error

**Comment by carlosehj** [comment]
> Não, desisti e assinei o FMP.

FMP brings adjusted closure?

**Comment by obafemisolo** [comment]
Concerning this, a simple view at Trading view privacy and security shows that a new session is created each time a request is made,  i tried using pickle to save and reuse login sessions but TvDatafeed did not expose its session attribute as expected, another work around is to change your IP address by simply restarting your router or airplane on/off will do, trading view block future login after receiving 5 logins from same IP Address within 24 hours.
hopefully in the future TvDataFeed will expose sessions attribute   or better still  save sessions and reuse them for new requests.
